src/featurizations/weavefeaturization.jl

```diff
@@ -3,12 +3,104 @@
 using ..ChemistryFeaturization.AbstractType: AbstractFeaturization
 using ..ChemistryFeaturization.FeatureDescriptor:
     AbstractAtomFeatureDescriptor, AbstractPairFeatureDescriptor
+using ..ChemistryFeaturization.Codec: DirectCodec
+using ..ChemistryFeaturization.Utils
 
 struct WeaveFeaturization <: AbstractFeaturization
     atom_features::Vector{<:AbstractAtomFeatureDescriptor}
+    bond_features::Vector{<:BondFeatureDescriptor}
     pair_features::Vector{<:AbstractPairFeatureDescriptor}
 end
 
+function WeaveFeaturization(default_atom_feature_list = ["symbol",
+                                                         "degree",
+                                                         "implicithconnected",
+                                                         "charge",
+                                                         "radical_electrons",
+                                                         "hybridization",
+                                                         "isaromatic",
+                                                         "hydrogenconnected"],
+                            default_bond_feature_list = ["bondorder", "isaromaticbond", "isringbond"])
+  species_feats = intersect(keys(Utils.SpeciesFeatureUtils.sfd_names_props), default_atom_feature_list)
+  
+  atoms = SpeciesFeatureDescriptor.(species_feats)
```

Comment:
This is a bit of a placeholder, since we would want to get both the species features and the atom features from different arrays. Would it be fine to split those out?

Reply:
I see what you mean how this is clunky. What do you mean by "split them out," though? Just split the species features and element features into separate lists?

Reply:
That's right, explicitly have different lists for different feature descriptors. We can concatenate the emergent encodings at either the `encode` step or the `featurize` step.